.github/actions/install/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Install Languages & Frameworks
```

Comment:
These kind of things would be a lot more useful if we could share them across providers.

Reply:
I shared it, via the `cp` command :)

Reply:
Duplication is eating away from our future productivity..

Reply:
Sure. I reduced duplication here. One step at a time.